pyodide_build/buildall.py

```diff
@@ -131,6 +131,15 @@ def make_parser(parser):
         default=common.TARGETPYTHON,
         help="The path to the target Python installation",
     )
+    parser.add_argument(
+        "--install-dir",
+        type=str,
+        nargs="?",
+        default="",
+        help="Directory for installing built host packages. Defaults to setup.py\
+default. Set to 'skip' to skip installation. Installation is needed if you want\
+to build other packages that depend on this one.",
```

Comment:
I meant,
```suggestion
        help=(
           "Directory for installing built host packages. Defaults to setup.py "
           "default. Set to 'skip' to skip installation. Installation is needed if you " 
           "want to build other packages that depend on this one."
        ),
```
+ maybe some black fmt, https://stackoverflow.com/a/2059025/1791279 I think that's also what's recommended in https://docs.python.org/3.7/tutorial/introduction.html#strings

In the above example there is a space missing anyway e.g. after `setup.py`